master/internal/rm/kubernetesrm/jobs_test.go

```diff
@@ -63,18 +64,74 @@ func TestGetNonDetPods(t *testing.T) {
 	ns2.On("List", mock.Anything, mock.Anything).Once().
 		Return(&k8sV1.PodList{Items: append(hiddenPods, expectedPods[1])}, nil)
 
-	p := jobsService{
+	js := jobsService{
 		podInterfaces: map[string]typedV1.PodInterface{
 			"ns1": ns1,
 			"ns2": ns2,
 		},
 	}
 
-	actualPods, err := p.getNonDetPods()
+	actualPods, err := js.getNonDetPods()
 	require.NoError(t, err)
 	require.ElementsMatch(t, expectedPods, actualPods)
 }
 
+func TestJobScheduledStatus(t *testing.T) {
+	pendingPod := k8sV1.Pod{
```

Comment:
i usually prefer declaring these closer to where they are used but with such a simple test i dont think it affects readability